### ISSUE TITLE
Unable to place limit-maker order

### ISSUE THREAD

**Issue by subhagato** [mpetrinidev/bnc-cli#1]
bnc --verbose spot new_order limit_maker --symbol ETH-BUSD --side SELL --quantity 0.005 -p 2485
2021-06-13 20:21:19.704384 [info] Initial values endpoint=https://api.binance.com/api/v3/order without_signature=False payload={'symbol': 'ETH-BUSD', 'side': 'SELL', 'type': 'LIMIT_MAKER', 'quantity': 0.005, 'price': 2485.0, 'newOrderRespType': 'ACK', 'recvWindow': 5000, 'timestamp': 1623615679703} method=POST headers={'User-Agent': 'bnc-cli  (https://github.com/mpetrinidev/bnc-cli)'}
2021-06-13 20:21:19.705525 [info] Signature created successfully
2021-06-13 20:21:19.706382 [info] Headers updated successfully at set_security headers={'User-Agent': 'bnc-cli  (https://github.com/mpetrinidev/bnc-cli)', 'X-MBX-APIKEY': '################################'}
2021-06-13 20:21:19.706481 [debug] Starting HTTP call to Binance API
2021-06-13 20:21:19.741829 [debug] Finishing HTTP call to Binance API
2021-06-13 20:21:19.742034 [info] Response details status_code=400 content-type=application/json;charset=UTF-8 x-mbx-uuid=696474fe-65be-4f2f-a893-8369283f536d x-mbx-used-weight=1 x-mbx-used-weight-1m=1 time-taken=0:00:00.033481
2021-06-13 20:21:19.742227 [debug] Processing response...
[exception] "Attempt to overwrite 'msg' in LogRecord"
[info] Submit an issue with --verbose details in https://github.com/mpetrinidev/bnc-cli/issues for help

**Comment by mpetrinidev** [comment]
There is an error on the symbol that you are using: `ETH-BUSD` is invalid.

The valid one is `ETHBUSD`

You can query Binance exchange information to get information about all symbols: [BNC Exchange info](https://github.com/mpetrinidev/bnc-cli/wiki/Market-Data#exchange-information)

---

On the other hand, there is an error related to the package when it tried to handle the proper error message. Gonna release a fix today about this.

Thanks

**Comment by mpetrinidev** [comment]
@subhagato new version has been released (version 0.0.3)

```shell
pip install --upgrade bnc
pip install --upgrade bnc_testnet
```

Thanks for taking the time to fill an issue
